app/design/frontend/Magento/blank/Magento_B2b/web/css/source/actions/_actions-select.less

```diff
@@ -0,0 +1,158 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+//
+//  Actions -> Action select
+//  _____________________________________________
+
+//
+//  Variables
+//  _____________________________________________
+
+@color-blue-pure: #007bdb;
+
+@action__height: 3.2rem;
+@action__width: 3.2rem;
+
+//  Triangle marker
+@button-marker-triangle__height: .5rem;
+@button-marker-triangle__width: .8rem;
+
+@button__border-color: @color-gray68;
+
+@action__border-color: @color-gray68;
+@action__active__border-color: @action__border-color;
+@action__hover__border-color: @action__border-color;
+
+@action-select__disabled__color: @text__color;
+@action-select__z-index: @z-index-5;
+
+@field-control__box-shadow: 0 0 3px 1px @color-blue3;
+@field-control__active__border-color: @color-blue-pure;
+@field-control__hover__border-color: @action__hover__border-color;
+
+@_dropdown__padding-right: @action__height;
+@_triangle__height: @button-marker-triangle__height;
+@_triangle__width: @button-marker-triangle__width;
+
+//  Action select have the same visual styles and functionality as native <select>
+.admin__action-group-wrap {
```

Comment:
I am not sure its correct. How this selector is going to fix Frontend styles? 
Sorry I cannot reproduce it really on my instance, maybe you can give some overview.